src/content/docs/apis/nerdgraph/examples/nerdgraph-historical-data-export.mdx

```diff
@@ -9,7 +9,7 @@ redirects:
 freshnessValidatedDate: never
 ---
 
-When running a [NRQL query](/docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-new-relics-query-language), you're restricted by [various limits](/docs/query-your-data/nrql-new-relic-query-language/get-started/rate-limits-nrql-queries), such as a limit on the number of data points returned in a response, and a query timeout. Our historical data export feature can be used to run a NRQL query that returns 200,000,000 data points in a response (instead of the usual [limit of 2000](/docs/query-your-data/nrql-new-relic-query-language/get-started/nrql-syntax-clauses-functions/#sel-limit)) and has no query timeout. The results are returned as one or more JSON files.
+When running a [NRQL query](/docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-new-relics-query-language), you're restricted by [various limits](/docs/query-your-data/nrql-new-relic-query-language/get-started/rate-limits-nrql-queries), such as a limit on the number of data points returned in a response, and a query timeout. Our historical data export feature can be used to run a NRQL query that returns 200,000,000 data points in a response (instead of the usual [limit of 5000](/docs/query-your-data/nrql-new-relic-query-language/get-started/nrql-syntax-clauses-functions/#sel-limit)) and has no query timeout. The results are returned as one or more JSON files.
```

Comment:
```suggestion
When running a [NRQL query](/docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-new-relics-query-language), you're restricted by [various limits](/docs/query-your-data/nrql-new-relic-query-language/get-started/rate-limits-nrql-queries), such as a limit on the number of data points returned in a response, and a query timeout. Our historical data export feature can be used to run a NRQL query that returns 200,000,000 data points in a response (instead of the usual [limit of 5,000](/docs/query-your-data/nrql-new-relic-query-language/get-started/nrql-syntax-clauses-functions/#sel-limit)) and has no query timeout. The results are returned as one or more JSON files.
```